test/helper.lua

```diff
@@ -99,8 +99,13 @@ function helpers.truncate_spaces(g)
     g.vinyl:truncate()
 end
 
-function helpers.is_expired_true()
-    return true
+function helpers.is_expired_true(task)
```

Comment:
In description you say "iterate_with" but here you change is_expired_true. It seems you fixed wrong function.

Reply:
Right, overlooked that it.